api_docs/markdowns/api/collections/collections.md

```diff
@@ -0,0 +1,229 @@
+- Response: `Datamodel`
+
+The `collection_id` is being retrieved through the collections list endpoint. It is also the name used for the collection on the data querying endpoint.
+For each field in the datamodel, a `kind` attributes specifies if a field is a dimension or a metric.
+
+[comment]: # (TODO: somehow provide a list of available dimensions and metrics. Maybe not exhaustive, so not necessarily dynamic)
+
+### Periods
+
+A period represents the datespan on which a timestamped collection is queried. The format used for one period is a list of two dates in a "YYYY-MM-DD" format.
+The dates on which a timestamped collection is available in the collections list endpoint through the *date_start* and *date_end* response keys.
+
+### Collections BQL Query
+
+A BQL Collection Query fhas the following format:
+```JSON
+{
+  "dimensions": [DIMENSIONS],
+  "metrics": [METRICS],
+  "filters": FILTER,
+  "sort": [SORTS]
+}
+```
+
+#### Fields
+
+Metrics and dimensions need to be prefixed with the collection from which they are queried.
+For a non-timestamped collection, the prefix is just the collection id:
+- `"{collection_id}.{field_slug}"`. For example: `"crawl.20180801.depth"`
+For a timestamped collection, one needs to additionally specify the period index (0-based):
+- `"{collection_id}.period_{idx}.{field_slug}"`. For example: `"search_console.period_0.count_clicks"`
+
+
+"Global" dimensions, available on all collections, can be specified as such. No need to prefix them:
+- `"{global_field_slug}"`. For example: `"url"`
+An exception to this is the date field, which will only need to be prefixed by the period index to group multiple timestamped collections together on a certain date span:
+- `"period_{idx}.date"`.
+
+
+In dimensions, metrics and filters, [[BQL Functions can be used. See documentation.;bql-functions]]
+
+#### Filters
+
+[[The BQL Filter documentation is available here.;bql-filter]]
+Fields are also to be prefixed on BQL Filters.
+
+#### Sort
+
+Each field on which we want to sort, can be specified through the format:
+```JSON
+{
+  "index": IDX,
+  "type": TYPE,
+  "order": ORDER
+}
+```
+where:
+- `IDX` is the 0-based index of the position of the field to sort
+- `TYPE` can be `"dimensions"` or `"metrics"` (defaults to `"dimensions"`)
+- `ORDER` can be `"asc"` or `"desc"` (defaults to `"asc"`)
+
+### Collections BQL Response
+
+The endpoint response is paginated and in JSON format. It is formalised as such:
+```JSON
+{
+  "previous": LINK_TO_PREVIOUS_PAGE,
+  "next": LINK_TO_NEXT_PAGE,
+  "page": PAGE_NUM,
+  "size": PAGE_SIZE,
+  "results": [
+    {
+        "metrics": [METRIC_VALUES],
+        "dimensions": [DIMENSION_VALUES]
+    },
+    ...
+  ]
+}
+```
+
+[comment]: # (TODO: add the count option information)
+
+## Examples
+
+First, a typical cURL pattern to use the collections API:
+
+```bash
+curl --request POST \
+  --url https://app.botify.com/api/v1/projects/{{username}}/{{project_slug}}/query?size=50 \
```

Comment:
I'd remove `--url`, and add quotes around the URL (not needed here, but needed as soon as one use `&count`)